.ci/ci_job_flags.sh

```diff
@@ -114,6 +114,24 @@ case "${CI_JOB}" in
 			;;
 	esac
 	;;
+"CC_CRI_CONTAINERD_CLOUD_HYPERVISOR"|"CC_SKOPEO_CRI_CONTAINERD_CLOUD_HYPERVISOR")
+	# This job only tests containerd + k8s
+	init_ci_flags
+	export CRI_CONTAINERD="yes"
+	export CRI_RUNTIME="containerd"
+	export KATA_HYPERVISOR="cloud-hypevisor"
+	case "${CI_JOB}" in
```

Comment:
Don't need this case statement; it will be either CC_CRI_CONTAINERD_CLOUD_HYPERVISOR or CC_SKOPEO_CRI_CONTAINERD_CLOUD_HYPERVISOR anyway.

Somewhat unrelated with this PR: I just realized we have configuration to test CLH + K8s + (containerd or cri-o) but we don't have to test "pure" CLH + containerd. Is it intentional?

Reply:
It's not intentional, what's the QEMU equivalent test that I missed? :-)